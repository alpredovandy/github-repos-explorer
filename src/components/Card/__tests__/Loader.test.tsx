import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import CardLoader from '@/components/Card/Loader';

describe('Test components: CardLoader', () => {
    const renderCardLoader = () => render(<CardLoader />);

    it('Snapshot', () => {
        const { container } = renderCardLoader();

        expect(container).toMatchSnapshot();
    });
});
