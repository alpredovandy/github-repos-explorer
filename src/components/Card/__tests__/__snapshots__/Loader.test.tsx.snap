// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test components: CardLoader > Snapshot 1`] = `
<div>
  <div
    class="grid grid-cols-3 gap-2"
  >
    <div
      class="col-span-3 skeleton h-6 w-full rounded"
    />
    <div
      class="col-span-1"
    />
    <div
      class="col-span-2"
    >
      <div
        class="skeleton h-6 w-full rounded mt-2"
      />
      <div
        class="skeleton h-4 w-full rounded mt-2"
      />
    </div>
  </div>
</div>
`;
